packages/lab/src/layout/SplitLayout/SplitLayout.css

```diff
@@ -0,0 +1,34 @@
+.uitkSplitLayout {
+  justify-content: space-between;
+}
+
+.uitkSplitLayout-separator {
+  --uitkFlexLayout-gap: calc(var(--uitk-spacing-unit) * var(--uitkFlexLayout-gap-multiplier) + var(--uitkFlexLayout-separator));
+}
+
+.uitkSplitLayout-separator > *:first-child::after,
+*:last-child::before,
```

Comment:
missing the first half of rule here, needs same prefix as rule abobe. As-is, has pretty wide reaching effects.